asterius/src/Asterius/Main.hs

```diff
@@ -103,6 +103,11 @@ parseTask args = case err_msgs of
              in if i >= 0 && i <= 2
                   then t {shrinkLevel = i}
                   else error "Shrink level must be [0..2]",
+          str_opt "thread-pool-size" $ \s t ->
```

Comment:
Don't forget to pass the `--thread-pool-size` option to `ahc-ld` as well. `ahc-ld` is called as the linker executable by `ahc`, so search for `--optl` in this module and see how other `ahc-ld` options are being passed.

Reply:
Good catch! Should be ok now.